site/app/templates/forum/searchResults.twig

```diff
@@ -24,8 +24,8 @@
                     </tr>
                     {% for post in thread.posts %}
                         <tr title="Go to post" class="cursor-pointer" onclick="window.location = '{{ post.post_link }}';" id="search-row-{{ post.count }}" class="hoverable">
-                            <td align="left"><pre class='pre_forum'>
-                                    <p class="post_content pre-wrap">
+                            <td align="left"><pre style="max-width: 67vw">
```

Comment:
Why the change to 67vw?

Reply:
Because 70vw is not working. Since we're probably going to do an Ajax search soon, I'm seeing this as a quick solution for something that will be removed soon!